### DISCUSSION TITLE
Rainbow's false advertising and subsequent failure to address this matter

### DISCUSSION THREAD

**Discussion by DayOfShah** [rainbow-me/rainbowkit#1960] [Q&A]
Approximately one week ago, on the Rainbow wallet app was a promotion for an NFT mint which the rainbow wallet described as free with a $0.02 (2 cents) gas fee. I minted two of these NFTs. Minting only required clicking mint, and no additional authorisations were required, and the transaction did not show me the actual cost of the mint. It was not until after having minted the NFTs that I saw that there was actually 0.0012 ETH fee that Rainbow charged without my knowledge or consent. 

It was extremely difficult to find any contact information for Rainbow which is suspicious in and of itself. However, I found this medium and reported the issue. In response, Rainbow advised me to contact Ethereum.

I responded to Rainbow asking why Rainbow was advising me to contact Ethereum for Rainbow's false advertising? Rainbow failed to respond so I followed up, and as I still have yet to hear back, I am writing here again for resolution of this matter. 

Return my funds now.

**Comment by DayOfShah** [comment]
I assure you, the longer Rainbow attempts to evade resolving this matter, and the more Rainbow attempt to evade accountability, the worse this repercussions will become.

**Reply by 0xmkh** [reply]
The team is looking at this and will give you a reply as soon as possible 🙏
